sdk/storage/Azure.Storage.Blobs/src/BlobClientOptions.cs

```diff
@@ -50,9 +56,15 @@ public enum ServiceVersion
         /// The <see cref="ServiceVersion"/> of the service API used when
         /// making requests.
         /// </param>
-        public BlobClientOptions(ServiceVersion version = LatestVersion)
+        /// <param name="customerProvidedKey">
+        /// The <see cref="CustomerProvidedKey"/> to be used when making requests.
```

Comment:
Nit - I think the doc comment here and on the property should mention something about the customer provided key to be used by the service for encrypting data to make it clear how this is used since the type/parameter names aren't great hints if you haven't seen that particular knob in the portal.

Reply:
And I guess everywhere else.  You could probably paste the same doc comment everywhere `<param name="customerProvidedKey">` shows up in a Find All across the solution.

Reply:
fixed.